examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -336,3 +337,29 @@ void emberAfThermostatClusterInitCallback(EndpointId endpoint)
 
     SetDefaultDelegate(endpoint, &delegate);
 }
+
+using chip::Protocols::InteractionModel::Status;
```

Comment:
I meant put this _inside_ the function, so the namespace import is function-local.

If we're going to do it for "everything after this point", just put it at the top of the file with the other "global" `using`.